felix/bpf/ut/bpf_prog_test.go

```diff
@@ -1245,6 +1251,12 @@ func withEgressQoSPacketRate() testOption {
 	}
 }
 
+func withEgressDSCP(value int8) testOption {
+	return func(o *testOpts) {
+		o.dscp = &value
```

Comment:
if -1 is invalid, you could have used -1 as invalid as well instead of using a pointer